### ISSUE TITLE
manual/javaGuide/main/tests/JavaTest.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#575]
- https://github.com/garbagetown/playdocja/blob/2.2.0/documentation/2.2.0/manual/javaGuide/main/tests/JavaTest.md

```
--- /Users/garbagetown/Desktop/2.1.5/manual/javaGuide/main/tests/JavaTest.md    2013-09-19 22:53:02.000000000 +0900
+++ /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/tests/JavaTest.md    2013-09-19 10:11:22.000000000 +0900
@@ -4,7 +4,7 @@

 ## Using JUnit

-The default way to test a Play 2 application is with [[JUnit| http://www.junit.org/]].
+The default way to test a Play application is with [JUnit](http://www.junit.org/).

 '''
 package test;
@@ -29,6 +29,15 @@
 }
 '''

+> **Note:** A new process is forked each time `test` or `test-only` is run.  The new process uses default JVM settings.  Custom settings can be added to `play.Project.settings` in `Build.scala`.  For example:  
+> '''
+> javaOptions in (Test) += "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=9998",
+> javaOptions in (Test) += "-Xms512M",
+> javaOptions in (Test) += "-Xmx1536M",
+> javaOptions in (Test) += "-Xss1M",
+> javaOptions in (Test) += "-XX:MaxPermSize=384M"
+> '''
+
 ## Running in a fake application

 If the code you want to test depends on a running application, you can easily create a `FakeApplication` on the fly:
@@ -52,4 +61,9 @@
 fakeApplication(inMemoryDatabase())
 '''

-> **Next:** [[Writing functional tests | JavaFunctionalTest]]
\ No newline at end of file
+> **Note:** Applications using Ebean ORM may be written to rely on Play's automatic getter/setter generation.  Play also rewrites field accesses to use the generated getters/setters.  Ebean relies on calls to the setters to do dirty checking.  In order to use these patterns in JUnit tests, you will need to enable Play's field access rewriting in test by adding the following to `play.Project.settings` in `Build.scala`:
+> '''
+> compile in Test <<= PostCompile(Test)
+> '''  
+
+> **Next:** [[Writing functional tests | JavaFunctionalTest]]

```

**Comment by garbagetown** [comment]
pull reqed at #680

**Comment by wm3** [comment]
merged #680
